test/ir_Fujitsu_test.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 Jonny Graham
+
+#include "IRsend.h"
+#include "IRsend_test.h"
+#include "ir_Fujitsu.h"
+#include "gtest/gtest.h"
+
+template<typename T, size_t size>
+::testing::AssertionResult ArraysMatch(const T (&expected)[size],
```

Comment:
I have to admit that I used stackoverflow :-)
I couldn't figure out how to change it so that the mismatch shows the value as HEX rather than decimal by the way. But when the unit tests pass it doesn't matter :-)